tests/framework/AnalyticModels/optimizing/plotWalk3d.py

```diff
@@ -0,0 +1,120 @@
+"""
+  Module for producing an animation of optimization histories
+  for 2D problems given pickled mesh grid data.  For examples
+  of the mesh grid data, see raven/tests/framework/AnalyticModels/optimizing/plot_functions.py.
+"""
+
+import pickle as pk
```

Comment:
in raven, the standard is camelBack... This is a utility method for RAVEN, so it should follow that standard.
In addition, the Module desclaimer is missing, (see one of the RAVEN modules)